service/src/main/java/bio/terra/workspace/common/utils/FlightUtils.java

```diff
@@ -276,9 +284,7 @@ public static boolean flightComplete(FlightState flightState) {
         "Testing flight {} completion; state is {}",
         flightState.getFlightId(),
         flightState.getFlightStatus());
-    return (flightState.getFlightStatus() == FlightStatus.ERROR
-        || flightState.getFlightStatus() == FlightStatus.FATAL
-        || flightState.getFlightStatus() == FlightStatus.SUCCESS);
+    return !flightState.isActive();
```

Comment:
Updated in the new version of Stairway (brought in through TCL change):

https://github.com/DataBiosphere/stairway/pull/137/files#r1540026406